[1.47.3] release notes (#529)
replicatedhq/kots.io: content/release-notes/1.47.3.md

---
date: 2021-07-23T10:30:14-08:00
linktitle: "1.47.3"
title: "1.47.3"
kubernetes: "1.18, 1.19, and 1.20"
---

{{<changes>}}
* Added support for uploading multiple files in the application config for repeatable config items.

> * Added support to remove a file from the application config when using the `file` item type.

Reviewer: ```suggestion
* Added support to remove a file from the application config when using the [`file` item](/reference/v1beta1/config/#file) type.
```

{{</changes>}}

{{<fixes>}}

> * Fixed a bug causing LicenseField template evaluations in the ConfigSpec file to fail.

Reviewer: ```suggestion
* Fixed a bug causing [LicenseFieldValue](/reference/template-functions/license-context/#licensefieldvalue) template evaluations in the [Config](/reference/v1beta1/config/) file to not render correctly.
```

* Fixed a bug that caused failed deployments to show as successful.

> * Fixed a bug that prevented helm charts from being deployed in multiple namespaces.

Reviewer: ```suggestion
* Fixed a bug that prevented [Helm charts](/reference/v1beta1/helmchart/) from being deployed in multiple namespaces.
```
